ci-scripts/unit-tests-sonar.sh

```diff
@@ -35,6 +35,18 @@ echo "Running unit tests and code coverage for product library"
 exec 5>&1
 output=$(ng test product --sourceMap --watch=false --code-coverage --browsers=ChromeHeadless | tee /dev/fd/5)
 
+echo "Running unit tests and code coverage for product-configurator library"
+exec 5>&1
+output=$(ng test product-configurator --sourceMap --watch=false --code-coverage --browsers=ChromeHeadless | tee /dev/fd/5)
```

Comment:
Add the lib in sonar configuration as well.

Reply:
Yes, added to sonar-project.properties